packages/e2e/integration/calendar_spec.ts

```diff
@@ -85,3 +89,28 @@ describe("Date Switcher", () => {
     });
   });
 });
+describe("Download ics file to Add To Calendar", () => {
+  it("downloads ics file", () => {
+    const downloadsFolder = Cypress.config("downloadsFolder");
+
+    cy.setClock(testDateLuxon.toMillis());
+
+    cy.initAdminApp().then((organization) =>
+      cy.updateFirestore(organization, [
+        "saul_with_extended_date.json",
+        "slots.json",
+      ])
+    );
+    cy.task("deleteFolder", downloadsFolder);
+    cy.visit([Routes.CustomerArea, saul.secretKey, "book_ice"].join("/"));
+    cy.getAttrWith("aria-label", "See past dates").click();
+    cy.getAttrWith("aria-label", "Book").first().click();
```

Comment:
Could you please use the aria label enums (like Chris did at some point, slot form I think...), so that we have that covered when merging this in.